src/diagnostics.rs

```diff
@@ -41,13 +44,25 @@ pub(crate) fn contrib_lint(pkg: &Package) -> Vec<lsp::Diagnostic> {
     visitor.diagnostics
 }
 
+pub(crate) fn no_influxdb_identifiers(
+    pkg: &Package,
+) -> Vec<(Option<String>, lsp::Diagnostic)> {
```

Comment:
So ah...just so I understand. This will return the error whenever the identifiers are used.

We use the identifiers in the prelude we send from the UI? Are we counting on the uri scope to mean the editor ignores errors tagged with the uri for the prelude.

Reply:
Ah yes. `PublishDiagnosticsParams` does return the uri. We are good! 😸